video/out/opengl/video.c

```diff
@@ -3107,7 +3107,7 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
         // Make the screen solid blue to make it visually clear that an
         // error has occurred
         float color[4] = {0.0, 0.05, 0.5, 1.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
Same, more or less. This is with the rendering-to-subrect feature in mind which I eventually want to implement.